website/cue/reference/components/sources/base/stdin.cue

```diff
@@ -5,12 +5,44 @@ base: components: sources: stdin: configuration: {
 		description: "Configures how events are decoded from raw bytes."
 		required:    false
 		type: object: options: {
+			avro: {
```

Comment:
Actually, I missed that this should be nested one more level. I.e. it should be at `decoding.codec.avro` rather than just `decoding.avro`.

Reply:
But it is one the same level as protobuf, json and syslog.
There is also a created entry in `decoding.codec.type.enum`

Reply:
Ah, you are right, the other codec options _are_ at that level.